test/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@
 cmake_minimum_required(VERSION 2.6 FATAL_ERROR)
```

Comment:
cmake_minimum_required(VERSION 3.9) ?

Reply:
I decide not to touch this file - just remove few `include_directory` commands.
`cmake_minimum_required` can be removed if we decide not to support standalone building of that directory